tests/JSONDependencyParserTest.php

```diff
@@ -23,9 +23,9 @@ class JSONDependencyParserTest extends TestCase
         }
         JSON;
 
-    public function __construct($name = null, array $data = [], $dataName = '')
```

Comment:
`__construct` is `final` and it takes a single argument.
I moved the body into `setUp()`